sql/gen-sql-config-docs.py

```diff
@@ -114,11 +119,17 @@ def generate_sql_configs_table_html(sql_configs, path):
 
 
 if __name__ == "__main__":
+    if len(sys.argv) != 2:
+        print("Usage: ./bin/spark-submit sql/gen-sql-config-docs.py <static|runtime>")
```

Comment:
You could just call the function twice instead of adding an argument considering that SQL configuration gen is pretty cheap. Let's fix it next time when we happen to touch here.

Reply:
got it